### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by ray8828** [GimSungwoo/WhaleID#1]
Hi, 
I've participated in the old  playground hampback challenge last year and my resnet seems not working well (just about 47%). I'm curious about the performance of this repo's method. Thanks.

Jiahui

**Comment by XingyuJinTI** [comment]
Hi, jiahui
Thanks for your question.
Pretrained resnet101 with freezing the whole CNN base gave the same performance with you.
To improve, we tried different fine-tuning strategy (i.e. we freeze less layers because only finetuning the FC-layers seems to under-fits the dataset which contains soo much classes.)  Only over 50% though, which I think is reasonable.

In terms of the recognition task, metric learning such as Siamese net could be more appropriate and the performance will see a boost. We will update the performance after all the experiments are done.

Xingyu
